services-manager/src/main/resources/application-example.yaml

```diff
@@ -0,0 +1,83 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+app:
+  version: '@project.version@'
+# While the library happens to be the same version as services, it should be declared here as a static name
+library:
+  version: ${app.version}
+
+manager:
+  # Defaults used internally to the yaml ONLY - these are not interpreted by any java application, just to minimise duplication of long filepaths etc.
+  default:
+    # Library includes relative to manager.root above (e.g. ../palisade-examples => palisade-services/../palisade-examples)
+    example-libs: &libraries
+    - ../Palisade-examples/example-library/target/example-library-${library.version}.jar
+    - ../Palisade-examples/hr-data-generator/target/hr-data-generator-${library.version}.jar
+
+  services:
+    audit-service:
+      jar: audit-service/target/audit-service-${app.version}-exec.jar
+      paths: *libraries
+      profiles:
+      - eureka
+      - example
+      log: audit-service-example.log
+      err: audit-service-example.err
```

Comment:
Final comment, promise!
Remove the duplicated jar, log, err fields. This should just enable the example stuff (so paths and profiles can stay)
And then ditto across other services here of course

Reply:
would you not want your example logs to go elsewhere?

Reply:
Fair enough, i didn't notice the different name.
I'd still remove the jar field

Reply:
And then therefore the app.version bit at the top

Reply:
So doing that change would mean that when starting the palisade example we need to have both services and example profiles. So I think for the sake of an extra line per service am happy to keep the jar's in here and decouple services (just loading the services) from the Example (loading everything needed for the example). Later on with the K8's smoke test stuff we will need to modify these to split out the eureka stuff as well anyway

Reply:
When starting the services for the example the services-manager uses the `example,debug` profiles. If this is removed then it would mean adding an extra profile to the startServices script.